selectors-4/Overview.bs

```diff
@@ -2256,20 +2256,9 @@ The Focus-Indicated Pseudo-class: '':focus-visible''</h3>
 		like clicking on the button with a mouse pointer.
 	</div>
 
-	<div class=advisement>
-		Page authors should follow these guidelines
-		when deciding whether to use '':focus'' or '':focus-visible''
-		to style the focused state of an element:
-
-		* If the element has “native” focus indicator behavior
-			(such as text fields or buttons),
-			use '':focus-visible''.
-		* Otherwise, if the element is emulating a text input,
-			or something else that is <em>intended</em> to receive keyboard interaction,
-			use '':focus''.
-		* Otherwise,
-			use '':focus-visible''.
-	</div>
+	This pseudo-class is intended to give developers the ability to style focus
+	selectively, in order to encourage visible, usable focus styles, and
```

Comment:
As a developer, personally I find the previous form of words more straightforward and easier to understand.  I'm struggling particularly with the word 'selectively'.  The purpose of `:focus-visible` is, as I understand it, to style the focus ring while not overriding the UA's discretion about whether to display it.  Therefore this mechanism intentionally doesn't give the developer control over that discretion, whereas I'd interpret 'selectively' in this text as implying that it does.